tests/Microsoft.Identity.Test.Unit/CoreTests/InstanceTests/GenericAuthorityTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@bgavrilMS These test rely on an online openidconnect server which I think is causing the build fail. I don't have enough permission in azure devops to see what is causing the build fail.